azure-pipelines.yml

```diff
@@ -106,6 +106,9 @@ jobs:
       cd DevGuideExamples\DCPS\Messenger
       perl integration_run_test.pl -ExeSubDir %SUBDIR_FOR_MESSENGER%
     displayName: Run Examples DCPS Messenger
+  - script: |
+      ! git clean -nd -e ext | cut -d ' ' -f 3 | xargs -I {} echo "##vso[task.logissue type=error]This file should be gitignored: '{}'" | exit 1
+    displayName: Check validity of .gitignore
```

Comment:
This doesn't necessarily need to run on the Windows builds.  Just having it on some builds will be helpful.  Ideally those would include optional features like Java and DDS Security.